src/Routes/RosSystemDetail/RosSystemDetail.js

```diff
@@ -51,36 +55,36 @@ class RosSystemDetail extends React.Component {
-                                <DescriptionListDescription>
-                                    { displayLastReported(reportDate) }
-                                </DescriptionListDescription>
-                            </DescriptionListGroup>
-                        </DescriptionList>
-                    </GridItem>
-                    <GridItem>
-                        <DescriptionList className='expanded-row' isCompact isHorizontal>
-                            <DescriptionListGroup>
-                                <DescriptionListTerm>State</DescriptionListTerm>
-                                <DescriptionListDescription>
-                                    <SystemState stateValue={ state }/>
-                                </DescriptionListDescription>
-                            </DescriptionListGroup>
-                        </DescriptionList>
-                    </GridItem>
-                    <GridItem>
-                        <ExpandedRow
-                            { ...{ cloudProvider, instanceType, idlingTime, inventoryId } }
-                        />
-                    </GridItem>
-                    <GridItem>
-                        <RecommendationRating system={ { ...{ rating, inventoryId } } } />
-                    </GridItem>
-                </Grid>
+
+                <Flex alignItems={{ default: 'alignItemsCenter' }}>
+                    <Flex grow={{ default: 'grow' }} className='ros-system-info'>
+                        <Flex flex={{ default: 'flex_4' }} direction={{ default: 'column' }}>
+                            <FlexItem>
+                                <DescriptionList className='expanded-row' isCompact isHorizontal>
+                                    <DescriptionListGroup>
+                                        <DescriptionListTerm>Last reported</DescriptionListTerm>
+                                        <DescriptionListDescription>
+                                            { displayLastReported(reportDate) }
+                                        </DescriptionListDescription>
+                                    </DescriptionListGroup>
+                                    <DescriptionListGroup>
+                                        <DescriptionListTerm>State</DescriptionListTerm>
+                                        <DescriptionListDescription>
+                                            <SystemState stateValue={ state }/>
+                                        </DescriptionListDescription>
+                                    </DescriptionListGroup>
+                                </DescriptionList>
+                                <ExpandedRow
+                                    { ...{ cloudProvider, instanceType, idlingTime, inventoryId } }
+                                />
+                                <RecommendationRating system={ { ...{ rating, inventoryId } } } />
```

Comment:
Is the extra object and spread required? I'd directly write it to the first object
```suggestion
                                <RecommendationRating system={ { rating, inventoryId } } />
```